### PR TITLE
Only unpremultiply extracted pixels if the render target is premultiplied

### PR DESCRIPTION
**Pull request by dev7355608**
##### Pre-Merge Checklist

- [x] Tests and/or benchmarks are included
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

File: packages/canvas-extract/src/CanvasExtract.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class CanvasExtract implements ISystem, IExtract
     };
 
     /** A reference to the current renderer */
-    public renderer: CanvasRenderer | null;
+    public renderer: CanvasRenderer;
```
[the remark this answers is not in the document]
Comment (bigtimebuddy) [reply packages/canvas-extract/src/CanvasExtract.ts:26]: The strict-types are a work in progress. Most of the other systems are in core which hasn't been converted to strict types yet.

[the remark this answers is not in the document]
Comment (dev7355608) [reply packages/canvas-extract/src/CanvasExtract.ts]: I see. I reverted the `| null` removal and handled the `Object is possibly 'null'` in `postrender`. Now there's no more complaints from `npm run types:strict`.

File: packages/core/src/textures/BaseTexture.ts [hunk 2]
```diff
@@ -72,32 +72,32 @@ export class BaseTexture<R extends Resource = Resource, RO = IAutoDetectOptions>
      * @member {PIXI.ALPHA_MODES}
      * @default PIXI.ALPHA_MODES.UNPACK
      */
-    public alphaMode?: ALPHA_MODES;
+    public alphaMode: ALPHA_MODES;
```
Comment (bigtimebuddy) [inline packages/core/src/textures/BaseTexture.ts:75]: One small request, could you create another PR for the optional members? Seems like a separate issue.

Comment (dev7355608) [reply packages/core/src/textures/BaseTexture.ts:75]: It triggered a bunch of strict type warning, so I included it here. But I can split it off.

Comment (bigtimebuddy) [reply packages/core/src/textures/BaseTexture.ts:75]: Yeah, it's helpful when we look back or have to revert to have PRs be single ideas.

**Pull request merged**